adapter/exporter/src/main/java/org/mobilitydata/gtfsvalidator/exporter/ProtobufNoticeExporter.java

```diff
@@ -177,6 +185,7 @@ private void outOfRangeNoticeToProto(String filename, String entityId, String fi
                 .setValue(rangeMinAsString)
                 .setAltValue(rangeMaxAsString)
                 .setAltEntityValue(actualValueAsString)
+                .clearAltEntityId()
```

Comment:
Good catch! As we said together, replace this by a call to `clear()` at the beginning of the chain to clear the reused internal builder